third_party/types/common-types.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
Accidentally checked in? Also, I suspect you will need to edit `tsconfig.json` to add `third_party` to the set of input files for it to be compiled automatically.